### PR TITLE
[16.0] [MIG] purchase_advance_payment: Migrate to version 16.0

### PR DESCRIPTION
**Pull request by sonhd91**

### CODE REVIEW

File: purchase_advance_payment/tests/test_purchase_advance_payment.py [hunk 1]
```diff
@@ -46,6 +46,21 @@ def setUpClass(cls):
         cls.purchase_order_1 = cls.env["purchase.order"].create(
             {"partner_id": cls.res_partner_1.id}
         )
+        cls.purchase_order_2 = cls.env["purchase.order"].create(
```
Comment (simahawk) [inline purchase_advance_payment/tests/test_purchase_advance_payment.py:50]: can you please add this after the call to super above? 
```
cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

**Review by simahawk — COMMENTED** [review]
Mere tech review, LG.

@LoisRForgeFlow can you help w/ a review?

File: purchase_advance_payment/tests/test_purchase_advance_payment.py (hunk 1 above)
Comment (sonhd91) [reply purchase_advance_payment/tests/test_purchase_advance_payment.py:50]: I just update the branch, please check again

**Review by yankinmax — APPROVED** [review]

**Review by santostelmo — APPROVED** [review]

**Pull request merged**